### DISCUSSION TITLE
Cascade delete from linked models

### DISCUSSION THREAD

**Discussion by bethel-m** [isar/isar#1605] [Help]
hello everyone, so i implemented a todo model in isar , where each todo task can have a taskgroup, while each taskgroup can have multiple tasks, i used links as stated in the documentation. But my issue now is how to implement cascade delete from the taskgroup, such that when a taskgroup is deleted every tasks linked to it is also deleted

**Comment by Jtplouffe** [comment]
You will have to implement this yourself, Isar v3 does not have any kind of side effect when deleting a record from a collection.
The only side effect is that links that were referencing the now deleted record will be null.
